### DISCUSSION TITLE
frame question

### DISCUSSION THREAD

**Discussion by Adwaith-Rajesh** [ParthJadhav/Tkinter-Designer#45] [Q&A] [answered]
If i have multiple frames, will it make a file for each frame

**Comment by ParthJadhav** [comment, marked as answer]
> If i have multiple frames, will it make a file for each frame

No, we are working on that feature. I'll let you know when it's available.

**Reply by yopsykochill** [reply]
lest say i have multiple frames created using the Designer, how do i switch between them using tkinter in python ?

**Reply by ParthJadhav** [reply]
Do you mean for conversion?
